prometheus-ksonnet/grafana/config.libsonnet

```diff
@@ -1,49 +1,35 @@
 {
   _config+:: {
-    admin_services+:: [
-      {
-        title: 'Grafana (Light)',
-        path: 'grafana',
-        params: '/?search=open&theme=light',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-      {
-        title: 'Grafana (Dark)',
-        path: 'grafana',
-        params: '/?search=open&theme=dark',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-    ],
-  },
+    // configure Grafana Lib based upon legacy configs below
+    rootUrl: $._config.grafana_root_url,
+    provisioningDir: $._config.grafana_provisioning_dir,
+    dashboardConfigMapCount: $._config.dashboard_config_maps,
 
-  grafana_config:: {
-    sections: {
-      'auth.anonymous': {
-        enabled: true,
-        org_role: 'Admin',
-      },
-      server: {
-        http_port: 3000,
-        router_logging: true,
-        root_url: $._config.grafana_root_url,
-      },
-      analytics: {
-        reporting_enabled: false,
-      },
-      users: {
-        default_theme: 'light',
+    grafana_ini+: {
```

Comment:
You can use `grafana.withGrafanaIniConfig()` here.

Reply:
yes!

Reply:
Actually, it doesn't work. That function is designed to be used at the root `grafana` level, as it adds a root _config here, and we're already inside _config.

Reply:
Maybe an overoptimization but you could to: `grafana_ini+: grafana.withGrafanaIniConfig(...).grafana_ini`